### ISSUE TITLE
Support 2FA

### ISSUE THREAD

**Issue by ehbush** [widewing/ha-toyota-na#22]
I'm a bit baffled at how folks are getting the integration to work, because for me - whether I attempt to login with my phone number or email, Toyota sends me a 2FA code (depending on the login medium, depends on where they send it ie. SMS vs email). Please please please please support a secondary modal which asks for the 2FA code in your Toyota HA Integration (much like for the Apple iCloud Integration, or the Alexa Integration, etc). I've been waiting soooooooo long for this and I think this is the only barrier left! 

See Discord for more information. I'm happy to be a guinea pig if you need anyone to help test.

**Comment by ehbush** [comment]
Just to ensure this issue has the latest information (for anyone else looking/watching and for posterity):

The issue for me, was I had an account and was using the Toyota Mobile App just fine. My profile was configured correctly in the Toyota App, but for some reason I needed to login to the web portal and complete my profile there (https://toyota.com/owners). Once I did that, the integration successfully authenticated & configured appropriately, without prompting for any 2FA.
